test/models/OverseasRefundPensionSchemeSpec.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import testdata.shortServiceRefunds.{overseasRefundPensionScheme, overseasSchemeProvider}
+import testdata.transfersIntoOverseasPensions.{
+  nonUkOverseasSchemeProvider,
+  transferPensionSchemeNonUK,
+  transferPensionSchemeUK,
+  ukOverseasSchemeProvider
+}
+import utils.TestUtils
+
+class OverseasRefundPensionSchemeSpec extends TestUtils {
```

Comment:
It's better to import TestUtils._ instead of extending. Inheritance is a very tightly coupling thing, coming from OOP. It's better if we like to be Functional to just import helper methods we want